bundles/framework/layerlist/view/LayerCollapse/StateHandler.js

```diff
@@ -47,7 +48,7 @@ export class StateHandler {
     }
     _getState () {
         return {
-            mutator: this.getMutator(),
+            mutator: this.mutator,
```

Comment:
Reusing the same object instead of always creating a new one. Otherwise memoization is useless.